src/app.js

```diff
@@ -277,24 +286,7 @@ var run = function () {
     }).appendTo('body')
   })
 
-  // ---------------- tabbed menu ------------------
-  $('#options li').click(function (ev) {
-    var $el = $(this)
-    selectTab($el)
-  })
-
-  var selectTab = function (el) {
```

Comment:
Hm, I actually can't remember removing this one... Would you maybe do this part, since you understand the debugging a bit better, so we don't mess up something here.